integrations/access/common/config.go

```diff
@@ -35,12 +35,15 @@ type PluginConfiguration interface {
 	GetTeleportClient(ctx context.Context) (teleport.Client, error)
 	GetRecipients() RawRecipientsMap
 	NewBot(clusterName string, webProxyAddr string) (MessagingBot, error)
+	GetRecipientsAreSchedules() bool
 }
 
 type BaseConfig struct {
 	Teleport   lib.TeleportConfig
 	Recipients RawRecipientsMap `toml:"role_to_recipients"`
 	Log        logger.Config
+	// RecipientsAreSchedules is used to mark wether the recipients of alerts are on-call schedules.
```

Comment:
Not sure I like introducing the seemingly opsgenie (and maybe pagerduty) specific concept of schedules into the `BaseConfig` / `BaseApp`. What about `bool UsersAsRecipients` or similar, that would indicate whether individual assigned reviewers on the access requests should be considered valid recipients? Could be `true` for Slack/Discord/similar, false for others.

Reply:
That sounds a lot clearer.